pytket/extensions/qujax/qujax_convert.py

```diff
@@ -123,6 +145,145 @@ def tk_to_qujax_symbolic(
             jnp.array([symbol_map[symbol] for symbol in c.op.free_symbols()])  # type: ignore
         )
 
-    return get_params_to_statetensor_func(
-        gate_name_seq, qubit_inds_seq, param_inds_seq, circuit.n_qubits
+    return gate_name_seq, qubit_inds_seq, param_inds_seq, circuit.n_qubits
+
+
+def tk_to_qujax_symbolic(
+    circuit: Circuit, symbol_map: Optional[dict] = None
+) -> qujax.UnionCallableOptionalArray:
+    """
+    Converts a tket circuit with symbolics parameters and a symbolic parameter map
+    into a function that maps circuit parameters to a statetensor.
+    Assumes all circuit gates can be found in qujax.gates.
+    Note that the behaviour of tk_to_qujax_symbolic(circuit)
+    is different to tk_to_qujax(circuit),
+    tk_to_qujax_symbolic will look for parameters in circuit.free_symbols()
+    and if there are none it will assume that none of the gates require parameters.
+    On the other hand, tk_to_qujax will work out which gates are parameterised
+    based on e.g. circuit.get_commands()[0].op.params
+
+    :param circuit: Circuit to be converted.
+    :type circuit: pytket.Circuit
+    :param symbol_map: dict that maps elements of circuit.free_symbols() (sympy)
+        to parameter indices.
+    :type symbol_map: Optional[dict]
+    :return: Function which maps parameters
+        (and optional statetensor_in) to a statetensor.
+        If the circuit has no parameters, the resulting function
+        will only take the optional statetensor_in as an argument.
+    :rtype: Callable[[jnp.ndarray, jnp.ndarray = None], jnp.ndarray]
+        or Callable[[jnp.ndarray = None], jnp.ndarray]
+        if no parameters found in circuit
+    """
+    return qujax.get_params_to_statetensor_func(
+        *tk_to_qujax_args_symbolic(circuit, symbol_map)
     )
+
+
+def print_circuit(
+    circuit: Circuit,
+    qubit_min: int = 0,
+    qubit_max: int = jnp.inf,  # type: ignore
+    gate_ind_min: int = 0,
+    gate_ind_max: int = jnp.inf,  # type: ignore
+    sep_length: int = 1,
+) -> List[str]:
+    """
+    Returns and prints basic string representation of circuit.
+
+    :param circuit: Circuit to be converted.
+    :type circuit: pytket.Circuit
+    :param qubit_min: Index of first qubit to display.
+    :type qubit_min: int
+    :param qubit_max: Index of final qubit to display.
+    :type qubit_max: int
+    :param gate_ind_min: Index of gate to start circuit printing.
+    :type gate_ind_min: int
+    :param gate_ind_max: Index of gate to stop circuit printing.
+    :type gate_ind_max: int
+    :param sep_length: Number of dashes to separate gates.
+    :type sep_length: int
+    :return: String representation of circuit
+    :rtype: List[str] each element represents a qubit
+    """
+    g, q, p, nq = tk_to_qujax_args(circuit)
+    return qujax.print_circuit(  # type: ignore
+        g, q, p, nq, qubit_min, qubit_max, gate_ind_min, gate_ind_max, sep_length
+    )
+
+
+def print_circuit_symbolic(
```

Comment:
Why is this a function in pytket-qujax? What is the benefit from having this in pytket-qujax and not only in qujax?